core/amber/src/main/resources/python_udf/nltk_sentiment_classify.py

```diff
@@ -0,0 +1,29 @@
+import pickle
+import pandas
+import texera_udf_operator_base
+
+
+class NLTKSentimentOperator(texera_udf_operator_base.TexeraMapOperator):
+
+	def __init__(self):
+		super(NLTKSentimentOperator, self).__init__()
+		self._model_file = None
+		self._sentiment_model = None
+
+	def open(self, args: list):
+		super(NLTKSentimentOperator, self).open(args)
+		model_file_path = args[2]
+		self._model_file = open(model_file_path, 'rb')
+		self._sentiment_model = pickle.load(self._model_file)
+		self._map_function = self.predict
```

Comment:
It seems like these belong in init?? Open() should contain variables that change through the lifecycle of the operator

Reply:
This cannot be put into `__init__()` because some necessary arguments are not known until `open()` is called. Here the necessary argument is the path of the model file. The file name is specified as an attribute of the Amber Python UDF operator, and the full path is generated on the Java side. @zuozhiw and I discussed this when defining the APIs for UDFs.